dnf5.spec

```diff
@@ -265,7 +265,7 @@ Development files for libdnf5-cli.
 %package -n perl5-libdnf5
 Summary:        Perl 5 bindings for the libdnf library
 License:        LGPL-2.1-or-later
-Provides:       perl(libdnf) = %{version}-%{release}
+# Provides:       perl(libdnf) = %{version}-%{release}
```

Comment:
Why did you only commented out the provide? May be it would be nice to know why you keep it if it is auto-generated.

Reply:
My mistake, I was locally testing. now the review passed and this is RFR